uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/ui/screens/habits/list/ListHabitsBehavior.kt

```diff
@@ -65,15 +64,7 @@ open class ListHabitsBehavior @Inject constructor(
                 commandRunner.run(CreateRepetitionCommand(habitList, habit, timestamp, value, newNotes))
             }
         } else {
-            screen.showCheckmarkDialog(
-                entry.value,
-                entry.notes,
-                timestamp.toLocalDate(),
-                timestamp.toDialogDateString(),
-                habit.color,
-            ) { newValue, newNotes ->
-                commandRunner.run(CreateRepetitionCommand(habitList, habit, timestamp, newValue, newNotes))
-            }
+            throw RuntimeException()
```

Comment:
No better way here I guess, but that shows we should make a better distinction between numerical and yes/no habits (different classes).